izpack-maven-plugin/src/main/java/org/izpack/mojo/IzPackNewMojo.java

```diff
@@ -187,14 +228,28 @@ public void execute() throws MojoExecutionException, MojoFailureException
         {
             throw new MojoExecutionException( "Failure", e );
         }
+    }
 
-        if (project.getPackaging().equals("izpack-jar"))
+    private boolean isSkipIzPack()
+    {
+        Properties userProperties = session.getUserProperties();
+        boolean skipIzPack = this.skipIzPack;
+        String skipIzPackStr = userProperties.getProperty("skipIzPack");
+        if (skipIzPackStr != null)
         {
-            project.getArtifact().setFile(jarFile);
+            skipIzPack = Boolean.parseBoolean(skipIzPackStr);
         }
```

Comment:
The `null` check could be left out as `Boolean.parseBoolean()` does also handle the `null` case as `false`

Reply:
If skipIzPack is specified on command line then only we will override or else we will be using what is specified under configuration or the default value. Adding this as a comment in the code.

Reply:
@reinhapa, is this ok?